terraform/e2e/main.tf

```diff
@@ -0,0 +1,61 @@
+# Copyright 2023 The Authors (see AUTHORS file)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module "jvs" {
+  source = "git::https://github.com/abcxyz/jvs.git//terraform/e2e?ref=main" # this should be pinned to the SHA desired
```

Comment:
Could you clarify the purpose of this e2e folder? is it for creating integration test and dev env? If so, it is better to directly use this module in our infrastructure code, otherwise we will end up rewrapping it there again, and any change to the source module, we will need to update multiple places.

If it is for creating an example for readers, readme is good enough?

Reply:
I agree this shouldn't be a _module_ for users to use. Instead, this should be an example of how to "hook up" jvs and the jira plugin. I recommend changing the name "e2e" => "example" and reference (not use the module) the example in the readme.

Reply:
Yeah, it is more of to provide an example on how to instantiate the environment. README is mostly good, it is just a bit awkward to describe how to figure out the service account to set as secret_accessor so I figure it is easier to show it in terraform.

Updated to example instead, let me know what you think.